internal/spdxlicense/license_test.go

```diff
@@ -35,6 +35,7 @@ func TestIDParse(t *testing.T) {
 			"CC-by-nc-3-de",
 			"CC-BY-NC-3.0-DE",
 		},
+                # the below few cases are NOT expected, however, seem unavoidable given the current approach
```

Comment:
@wagoodman you might have another language on the brain. I think this should be `//`

Reply:
I like to throw in breaking code into other folks PRs as a new hobby of mine...  couldn't get past you!